### ISSUE TITLE
Cant see images in LMDB

### ISSUE THREAD

**Issue by madhavajay** [weiliu89/caffe#898]
### Issue summary
I have generated datasets both of COCO and of VOC (with voc using the exact commands on the SSD Caffe tutorial which work flawlessly (so im assuming they are correct).

However, when I try to open the lmdb training databases and view the images it doesnt work.
I am using code like this:
http://shengshuyang.github.io/hook-up-lmdb-with-caffe-in-python.html

The result of datum.ParseFromString(value) is nothing.
However value is a lot of binary data.
I have tried saving the binary data out to jpg and its unreadable by any application.

I read somewhere in some of the Caffe threads this was an issue with numpy and lmdb and 32bit / 64bit floating point storage or something, but it was fixed in the mainline.

Does anyone else have this problem? It doesn't feel good that I cant view the images in the lmdb with the same Caffe code that will be training them.

### Steps to reproduce
Generate the VOC dataset as per the instructions.
Open a jupyter notebook and run the code above to see the images.

### Your system configuration
Operating system: Ubuntu 16.04
Compiler: gcc? not sure
CUDA version 9.0 ubuntu1604_9.0.176-1_amd64.deb
CUDNN version 7.0 libcudnn7_7.0.5.15-1+cuda9.0_amd64.deb
BLAS: OpenBlas
Python or MATLAB version (for pycaffe and matcaffe respectively): Python 2.7

**Comment by geoffzhang** [comment]
Hi, I have the same question, do you solve it?

**Comment by madhavajay** [comment]
No, I didnt solve it and I havent tried to fix it since, but I might take another look if I next fire up Caffe SSD again. I am hoping I can just move to TensorFlow as Caffe is kinda dying off.
Its a pity because LMDB is supposed to be faster than TFRecord.
